src/app/hooks/useGlean.ts

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { useEffect } from "react";
+import Glean from "@mozilla/glean/web";
```

Comment:
Do we need something like https://www.mozilla.org/en-US/privacy/websites/data-preferences/ ?

Reply:
@rhelmer if GA doesn't need it, Glean doesn't need it. If you're product needs it, then you can/should add it. You might want to double check with legal, but there's no Glean-specific requirement around this.